modules/game/src/main/Query.scala

```diff
@@ -26,7 +26,10 @@ object Query:
 
   val mate: Bdoc = status(Status.Mate)
 
-  def draw(u: UserId): Bdoc = user(u) ++ finished ++ F.winnerId.$exists(false)
+  def draw(u: UserId): Bdoc =
+    user(u) ++ finished ++ F.winnerId.$exists(false) ++ $doc(
+      F.status $in List(Status.Stalemate.id, Status.Draw.id)
+    )
```

Comment:
Could the `finished` condition be removed if we're checking for explicit status ids?

Reply:
Good catch, `finished` is redundant here. Tested and removed in latest commit.